src/coreclr/src/vm/peimage.h

```diff
@@ -120,7 +120,8 @@ class PEImage
     };
 
     static PTR_PEImage FindById(UINT64 uStreamAsmId, DWORD dwModuleId);
-    static PTR_PEImage FindByPath(LPCWSTR pPath);
+    static PTR_PEImage FindByPath(LPCWSTR pPath,
+                                  BOOL lookInBundle = true);
```

Comment:
Nit: I would call the argument `isInBundle` to match the name we are using everywhere else